.github/workflows/pypi.yml

```diff
@@ -74,34 +62,11 @@ jobs:
           python-version: '3.12'
 
       - name: Install dependencies
-        run: pip install pytest
-
-      - name: Print current directory contents (Linux/macOS)
-        if: runner.os != 'Windows'
-        run: |
-          echo "==================== Current Directory Contents ===================="
-          pwd
-          ls -al
-          echo "==================================================================="
-
-      - name: Print current directory contents (Windows)
-        if: runner.os == 'Windows'
-        run: |
-          echo "==================== Current Directory Contents ===================="
-          echo "Full Path: %cd%"
-          dir
-          echo "==================================================================="
+        run: pip install . pytest
```

Comment:
You must install the built wheel or the local directory. Tests should not be run without installing first. Since cibuildwheel already runs your tests, I assume it's better to build here.